app/views/comparisons/electricity_targets/_table.html.erb

```diff
@@ -1,27 +1 @@
-<%= component 'comparison_table',
-              report: report, advice_page: advice_page, table_name: table_name, index_params: index_params,
-              headers: headers, colgroups: colgroups, advice_page_tab: advice_page_tab do |c| %>
-  <% results.each do |row| %>
-    <% c.with_row do |r| %>
-      <% r.with_school school: row.school %>
-      <%= r.with_var change: true,
-                     val: row.current_year_percent_of_target_relative, unit: :relative_percent %>
-      <%= r.with_var change: true,
-                     val: row.current_year_unscaled_percent_of_target_relative, unit: :relative_percent %>
-      <%= r.with_var val: row.current_year_kwh, unit: :kwh %>
-      <%= r.with_var val: row.current_year_target_kwh, unit: :kwh %>
-      <%= r.with_var val: row.unscaled_target_kwh_to_date, unit: :kwh %>
-      <%= r.with_var val: row.tracking_start_date, unit: :date %>
-    <% end %>
-  <% end %>
-  <% c.with_footer do %>
-  <tr>
-    <td colspan="<%= headers.count %>">
-      <p>
-        <strong><%= t('analytics.benchmarking.content.footnotes.notes') %></strong>
-      </p>
-      <%= t('analytics.benchmarking.configuration.column_heading_explanation.last_year_definition_html') %>
-    </td>
-  </tr>
-  <% end %>
-<% end %>
+<%= render partial: 'comparisons/shared/targets/table', locals: local_assigns %>
```

Comment:
like the local_assigns, would be good to do this on the others

Reply:
Good point, I'll add them to the PR 👍